app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if "mv" in command:
```

Comment:
You must split command to part.
Because for example:
command = "hellomv" will return true